map/src/menu/actions/TrackActions.jsx

```diff
@@ -53,6 +53,7 @@ const TrackActions = forwardRef(({ track, setDisplayTrack, setOpenActions }, ref
                 <Paper id="se-track-actions" className={styles.actions}>
                     {ctx.gpxFiles[track.name]?.showOnMap ? (
                         <MenuItem
+                            id={'se-hide-from-map-action'}
```

Comment:
Avoid using {} when just string passed to attributes. id="se-hide-from-map-action"